content/lizheming/drone-wechat/index.md

```diff
@@ -0,0 +1,91 @@
+---
+date: 2018-01-07T00:00:00+00:00
+title: Wechat
+author: lizheming
+tags: [ notifications, chat ]
+repo: lizheming/drone-wechat
+logo: wechat.svg
+image: lizheming/drone-wechat
+---
+
+The Wechat plugin posts build status messages to your account. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipline:
+  wechat:
+    image: lizheming/drone-wechat
+    sckey: xxxxxx
+    title: "plugin notification"
+    message: >
+      {%if success %}
+        build {{build.number}} succeeded. Good job.
+      {% else %}
+        build {{build.number}} failed. Fix me please.
+      {% endif %}
+```
+
+# Praameter Reference
+
+sckey
+: SCKEY get from [ServerChan](http://sc.ftqq.com)
+
+title
+: Notification title
+message
+: Notification body message, support markdown
+
+# Secret Reference
+
+server\_chan\_key
```

Comment:
Normally secrets are prefixed with the plugin name, e.g. `wechat_server_chan_key` or `wechat_sckey`

Reply:
OK, I'll change it later, thank you very much.